packages/core/src/events/TapEvent.ts

```diff
@@ -322,19 +314,36 @@ const applyEvent = (target: Global): void => {
     }
   }, true);
 
+  /**
+   * Listen to `keydown` event on the target
+   * Use this to fire tap event, if `enter` is clicked and prevent default if ' ' is clicked
```

Comment:
Not sure it's helpful to just show ' '. It's quite hard to read. We should prbably use 'pressed' instead of 'clicked' for key events.
```suggestion
   * Use this to fire tap event, if `ENTER` is pressed and prevent default if 'SPACE ' is pressed.
```

Reply:
OK